features/support/env.rb

```diff
@@ -43,16 +43,18 @@
 else # else create driver instance for desktop browser
   begin
     chromedriver_path = File.join(File.absolute_path(''),"chromedriver")
-    Selenium::WebDriver::Chrome.driver_path = chromedriver_path
+    Selenium::WebDriver::Chrome::Service.driver_path = chromedriver_path
```

Comment:
You change this file for any problem or for what? Works?

Reply:
To solve deprecated warnings

Reply:
Ok, perfectly.  We apply in our boostrap :+1: